.github/workflows/wheels.yml

```diff
@@ -3,6 +3,7 @@ name: Build and Release
 on:
   release:
     types: [published]
+  pull_request:
```

Comment:
@kafonek before trying the deploy, is there a reason that we trigger the wheel build on PRs?

Reply:
My goal here is to raise some of the errors we've been seeing before getting to a release action.

Reply:
Might be a bit overkill to run it on every PR, since we only need to check on PRs immediately before a release. Maybe we could use `[workflow_dispatch`](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow) instead?

Reply:
I'm open to using that and manually kicking off actions if you want to add me to the repo with write access 🙏 I feel bad that there's so many release action failures based on some CI PR's I've put in.

Reply:
I really appreciate the work you put into this. Wish it wasn't so hard to verify. Unfortunately, I don't have control over read/write access to the repo, but I think @dmonad can help with this.